.travis/deploy.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+TAG="$DEPLOY_SERVER/$DEPLOY_PROJECT/$DEPLOY_IMAGE:latest"
```

Comment:
Nit but I think the idea of the org as the project is an OpenShift concept. Is there a different name for "the second thing between slashes in the image tag" that we could use?

Reply:
namespace?